<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="GameFrame">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(120, 120, 120);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="ExplanationLabel">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>461</width>
          <height>331</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(222, 221, 218);
border-width: 3px;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Welcome  to our Dippid-FruitBasket-Game.
 Please choose the port your device is going to use and click &quot;Connect&quot;. 
 After you click start you will be able to control the basket at the bottom by rotation your device left and right. You can change your basket by using Button 1 and 3 on your device. Try  to catch as many fruits as you can with the correct basket.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="PlayerIcon">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>440</y>
          <width>81</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3,1,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PortLineEdit">
         <property name="text">
          <string>5700</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ConnectButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="StartButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>CurrentScore:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="ScoreScreen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
